homeassistant/components/bang_olufsen/media_player.py

```diff
@@ -681,7 +687,7 @@ async def async_play_media(
                     )
 
             except ApiException as error:
-                _LOGGER.error(json.loads(error.body)["message"])
+                raise HomeAssistantError(json.loads(error.body)["message"]) from error
```

Comment:
we should add a short description which includes the originated error message as a placeholder like "an error occurred while .... : {error_message}"

Reply:
Yes, I've improved the error message now.